valhalla/jawn/src/lib/stores/ScoreStore.ts

```diff
@@ -169,6 +166,92 @@ export class ScoreStore extends BaseStore {
     return ok(rowContents.data);
   }
 
+  public async putScoreIntoClickhouse(newVersion: {
+    id: string;
+    version: number;
+    provider: string;
+    score: Score;
+  }): Promise<Result<InsertRequestResponseVersioned, string>> {
```

Comment:
Why don't we just reuse the `putScoresIntoClickhouse` function here? We are copying the entire implementation

Reply:
@chitalian make sense! fixed this